### DISCUSSION TITLE
suppress message for creating tmp folders

### DISCUSSION THREAD

**Discussion by Dronakurl** [Robitx/gp.nvim#95] [Q&A]
Somehow, I regularly get the message
`Gp: creating directory /tmp/gp_images`
on nvim startup. How can I suppress these messages?

**Comment by Robitx** [comment]
@Dronakurl what version are you using?

**Comment by Dronakurl** [comment]
It is version 2.4.4. Sorry for the late reply. But I didn't notice the
problem for a while now.

Am Di., 23. Jan. 2024 um 20:00 Uhr schrieb Tibor Schmidt <
***@***.***>:

**Reply by Robitx** [reply]
@Dronakurl I was asking about the version, because this has been fixed since 2.3.11. :slightly_smiling_face:

**Comment by Dronakurl** [comment]
Thank you!!
